clients/privacy-center/config/config.json

```diff
@@ -1,6 +1,8 @@
 {
   "title": "Take control of your data",
   "description": "When you use our services, you’re trusting us with your information. We understand this is a big responsibility and work hard to protect your information and put you in control.",
+  "description_subtext": [],
+  "addendum": ["This is text that appears after the privacy options."],
```

Comment:
Nit: this default text will appear on privacy centers by default... So you should set this to be empty 😛